tests/manage/test_ocs_basics.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+from ocs import ocp, constants
+from ocsci.config import ENV_DATA
+from ocsci.testlib import tier1, ManageTest
+from tests import helpers
+
+import pytest
+
+log = logging.getLogger(__name__)
+
+
+POD = ocp.OCP(kind='Pod', namespace=ENV_DATA['cluster_namespace'])
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    def finalizer():
+        teardown()
+
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment for the test
```

Comment:
Can we be more specific here? Let's describe what we are doing to set up the environment.